snippets/base/management/commands/generate_bundles.py

```diff
@@ -92,12 +105,21 @@ def handle(self, *args, **options):
                             default_storage.delete(filename)
                         continue
 
-                    data = [job.render() for job in bundle_jobs]
+                    data = []
+                    channel_job_ids = list(channel_jobs.values_list('id', flat=True))
+                    for job in bundle_jobs:
```

Comment:
Slightly re-engineered this bit to make sure that in the -unlikely- event that a Job targets both `nightly` and `release` we render it normally and not with `always_eval_to_false=True`

Reply:
Also updated test